test/livebook/intellisense_test.exs

```diff
@@ -1089,6 +1086,21 @@ defmodule Livebook.IntellisenseTest do
                )
     end
 
+    test "completion for struct keys inside struct ignores `__exception" do
+      {binding, env} = eval(do: nil)
+
+      refute [
+               %{
+                 label: "__exception__"
+               }
+             ] =
+               Intellisense.get_completion_items(
+                 "%Livebook.IntellisenseTest.MyStruct{my",
```

Comment:
To be clear, you need to declare an exception, with no fields, and then try to autocomplete this:

```suggestion
                 "%Livebook.IntellisenseTest.MyException{",
```

Reply:
oops! I think my comment above still stands though

Reply:
Yes, the way the test passes is good still. :)

Reply:
Fixed with `%ArgumentError{`